Extension/src/nativeStrings.json

```diff
@@ -481,5 +481,6 @@
     "expand_selection": "Expand selection (to enable 'Extract to function')",
     "file_not_found_in_path2": "\"{0}\" not found in compile_commands.json files. 'includePath' from c_cpp_properties.json in folder '{1}' will be used for this file instead.",
     "copilot_hover_link": "Generate Copilot summary",
-    "browse_path_not_found": "Unable to index files from non-existent folder: {0}"
+    "browse_path_not_found": "Unable to index files from non-existent folder: {0}",
+    "license_terms": "The C/C++ Extension may be used only with Microsoft Visual Studio, Visual Studio for Mac, Visual Studio Code, Azure DevOps, Team Foundation Server, and successor Microsoft products and services to develop and test your applications."
```

Comment:
I think "Extension" should be "extension".

Reply:
I was matching the capitalization used in [the existing license](https://github.com/microsoft/vscode-cpptools/blob/main/RuntimeLicenses/cpptools-LICENSE.txt#L1).

> All VS Marketplace extensions, including this C/C++ Extension for Visual Studio...

Reply:
Given that we're localizing this anyway, I'll go ahead and change it.